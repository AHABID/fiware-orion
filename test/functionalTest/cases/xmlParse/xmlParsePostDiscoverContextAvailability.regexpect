+++++ Registration +++++
HTTP/1.1 200 OK
Content-Length: 141
Content-Type: application/xml
Date: REGEX(.*)

<?xml version="1.0"?>
<registerContextResponse>
  <duration>PT1H</duration>
  <registrationId>REGEX([0-9a-f]{24})</registrationId>
</registerContextResponse>
+++++ Registration Done +++++
+++++ Discovery +++++
HTTP/1.1 200 OK
Content-Length: 904
Content-Type: application/xml
Date: REGEX(.*)

<?xml version="1.0"?>
<discoverContextAvailabilityResponse>
  <contextRegistrationResponseList>
    <contextRegistrationResponse>
      <contextRegistration>
        <entityIdList>
          <entityId type="Room" isPattern="false">
            <id>ConferenceRoom2</id>
          </entityId>
          <entityId type="Room" isPattern="false">
            <id>OfficeRoom2</id>
          </entityId>
        </entityIdList>
        <contextRegistrationAttributeList>
          <contextRegistrationAttribute>
            <name>temperature</name>
            <type>degree</type>
            <isDomain>false</isDomain>
          </contextRegistrationAttribute>
        </contextRegistrationAttributeList>
        <providingApplication>http://localhost:1028/application</providingApplication>
      </contextRegistration>
    </contextRegistrationResponse>
  </contextRegistrationResponseList>
</discoverContextAvailabilityResponse>
+++++ Discovery Done +++++
