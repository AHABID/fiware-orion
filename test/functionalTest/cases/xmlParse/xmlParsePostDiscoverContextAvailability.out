+++++ Registration +++++
HTTP/1.1 200 OK
Content-Length: 141
Content-Type: application/xml
Date: Sun, 01 Jun 2014 17:45:38 GMT

<?xml version="1.0"?>
<registerContextResponse>
  <duration>PT1H</duration>
  <registrationId>538b66c20c86a938d2dc5370</registrationId>
</registerContextResponse>
+++++ Registration Done +++++
+++++ Discovery +++++
HTTP/1.1 200 OK
Content-Length: 904
Content-Type: application/xml
Date: Sun, 01 Jun 2014 17:45:38 GMT

<?xml version="1.0"?>
<discoverContextAvailabilityResponse>
  <contextRegistrationResponseList>
    <contextRegistrationResponse>
      <contextRegistration>
        <entityIdList>
          <entityId type="Room" isPattern="false">
            <id>ConferenceRoom2</id>
          </entityId>
          <entityId type="Room" isPattern="false">
            <id>OfficeRoom2</id>
          </entityId>
        </entityIdList>
        <contextRegistrationAttributeList>
          <contextRegistrationAttribute>
            <name>temperature</name>
            <type>degree</type>
            <isDomain>false</isDomain>
          </contextRegistrationAttribute>
        </contextRegistrationAttributeList>
        <providingApplication>http://localhost:1028/application</providingApplication>
      </contextRegistration>
    </contextRegistrationResponse>
  </contextRegistrationResponseList>
</discoverContextAvailabilityResponse>
+++++ Discovery Done +++++
